enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -380,9 +387,15 @@ def _create_redemption(
         ledger_transaction.state = TransactionStateChoices.PENDING
         ledger_transaction.save()
 
+        try:
+            if self.geag_fulfillment_handler().can_fulfill(ledger_transaction):
+                self.geag_fulfillment_handler().fulfill(ledger_transaction)
+        except Exception as exc:
+            self.rollback_transaction(ledger_transaction)
+            raise exc
```

Comment:
My only thought here is to make the exception handling more specific - we might not want to rollback the transaction depending on what point the fulfillment flow got to.  Doesn't have to block this PR.

Reply:
This is also the point at which we might take the path of returning a PENDING transaction all the way back to the enterprise-access -> frontend for async processing, right?

Reply:
@iloveagent57 yep, agreed lemme noodle this further as the final iteration comes together.

@pwnage101 re: our IRL discussion earlier, we are proceeding with this being fairly synchronous for now.